src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FlexContainer } from '../FlexContainer';
+import { FilterRow } from './FilterRow';
+import { FilterProps } from './Filters.types';
+import { OptionPropType } from './Select/Select.types';
+
+const Filters: React.FC<FilterProps> = ({ options }) => (
+  <FlexContainer flexDirection="column">
+    {options.map(({ uuid, ...rest }, index) => (
+      <FilterRow key={uuid} isFirstRow={index === 0} uuid={uuid} {...rest} />
+    ))}
+  </FlexContainer>
+);
+
+export default Filters;
+
+const RowPropType = PropTypes.exact({
+  uuid: PropTypes.string.isRequired,
```

Comment:
I wouldn't call it `uuid` since `uuid` have strict format. I would call it `name` or just `id`.

Reply:
I pick `id` to have it clear it should be unique for further remove handling